features/support/env.rb

```diff
@@ -1,4 +1,5 @@
 require 'cucumber/rails'
+require "factory_girl_rails"
 ActionController::Base.allow_rescue = false
 
```

Comment:
You can add the FactoryGirl matchers and shorten the step def'w by omitting `FactoryGirl` when you call that in your definitions.